ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/SendTransactionHandler.java

```diff
@@ -99,20 +102,20 @@ public void handle(final RoutingContext context, final JsonRpcRequest request) {
 
   private void sendTransaction(
       final Transaction transaction,
-      final HttpServerRequest httpServerRequest,
+      final RoutingContext routingContext,
       final JsonRpcRequest request) {
     final TransactionTransmitter transmitter =
-        createTransactionTransmitter(transaction, httpServerRequest, request);
+        createTransactionTransmitter(transaction, routingContext, request);
     transmitter.send();
   }
 
   private TransactionTransmitter createTransactionTransmitter(
       final Transaction transaction,
-      final HttpServerRequest httpServerRequest,
+      final RoutingContext routingContext,
       final JsonRpcRequest request) {
 
     final SendTransactionContext context =
-        new SendTransactionContext(httpServerRequest, request.getId(), transaction);
+        new SendTransactionContext(routingContext, request.getId(), transaction);
```

Comment:
don't think we need this in the context, it's only needed by the transmitter. just pass it into the transmitter constructor?

Reply:
For now the context will remain until we've worked out how errors are propogated BACK to the caller.